docs/DeveloperGuide.md

```diff
@@ -2,33 +2,82 @@
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+### Class diagram for v2.0
+![Class Diagram](assets/classdiagram.svg)
+
+We have 4 packages in total: Control, Entity, Seedu.Duke(boundary) and exception. 
+Control classes deal with the database of entity objects directly, return the wanted result after processing the database.  
+Entity classes include facilities that are available to be navigated to.
+Boundary classes take in user input and process the user input to pass to control class.
+Exception handles common exceptions such as invalid input exception etc to make sure the system runs smoothly.
+
+
+### Sequence digram for v2.0
+### Sequence digram for listAllLocations
+![Sequence Digram of listAllLocations](https://github.com/AY2021S2-CS2113-T10-2/tp/blob/master/docs/assets/Sequence%20Digram.JPG)
```

Comment:
seems like you could include your user stories here and likewise i think you did not link your image properly so it is not shown on the developer's guide